src/main/java/com/gw/exercises/task1/helloName.java

```diff
@@ -0,0 +1,33 @@
+package com.gw.exercises.task1;
+
+public class helloName {
+
+    // All className we need to start Capitalised
+    //it suitable to use "_" and numbers
+    // i can to start naming variable from $ and _
+    // camelCaseUsedToSeparateWords
+
+
+    //variables
+    // byte    -128 /  127 - 1 byte
+    // short   -32768 /  32767 - 2 byte
+    // int     -2147483648 / 2147483648  - 4 byte
+    // long    -9223372036854775808 / 9223372036854775808 - 8 byte
+    // float   -3.4E+38  / 3.4E+38  - 4 byte
+    // double  -1.7E+308 / 1.7E+308 - 8 byte
+    // char    0 - 65536 - 2 byte
+    // boolean  1bit but in real compilers varies
+    // void  empty value
```

Comment:
The same thing - what was the reason that you wrote these comments?

Reply:
Just a note for myself. Will discard it.